build.gradle

```diff
@@ -95,9 +96,8 @@ dependencies {
         exclude group: 'xml-apis'
     }
 
-    // FindBugs dependencies
-    compile 'com.google.code.findbugs:findbugs:3.0.1'
-    compile 'xml-apis:xml-apis:1.0.b2' // this version is needed by FindBugs on runtime
```

Comment:
And now it isn't?

Reply:
I've run integration tests and it runs fine. SpotBugs has upgraded to JDK 8 and I think that's why we don't need this backward dependency.